plugins/launchtube/package.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
My initial thinking is let push it to npm separate from this project and let users use it when needed.

Is that feasible?

We can create example in examples dir where stellar + launch tube is used

Reply:
I definitely think that is the approach to go with.